changelogs/fragments/65304-fix_zabbix_host_inventory_mode_key_error.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fix to inventory_mode key error for occur in zabbix 4.4.1 or more (https://github.com/ansible/ansible/issues/65304).
```

On a second thought, this is at least missing module name

```suggestion
  - zabbix_host - fixed inventory_mode key error, which occurs with Zabbix 4.4.1 or more (https://github.com/ansible/ansible/issues/65304).
```

---

Thank you for the pointed out.
Fixed this.